soc/arm/infineon_xmc/4xxx/soc.c

```diff
@@ -0,0 +1,33 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright (c) 2020 Linumiz
+ * Author: Parthiban Nallathambi <parthiban@linumiz.com>
+ *
+ */
+
+#include <kernel.h>
+#include <init.h>
+#include <soc.h>
+
+#define PMU_FLASH_WS		(0x3U)
+
+void z_platform_init(void)
+{
+	u32_t temp;
+
+	/* Enable FPU */
+	SCB->CPACR |= ((3UL << 10*2) | (3UL << 11*2));
```

Comment:
Can we use CPACR defines from cmsis.h so this isn't all magic numbers.  Should this also be wrapped in a CONFIG_FLOAT

Reply:
Sure, adapted it. Thanks.